dap-variables.el

```diff
@@ -164,11 +178,15 @@ Return the result."
       (goto-char (point-min))
 
       (save-match-data
-        (while (re-search-forward "${\\([^}]*\\)}" nil t)
-          (let ((var (match-string 1)))
-            (replace-match
-             (with-current-buffer old-buffer
-               (dap-variables-expand-variable var))))))
+        (while (re-search-forward
+                "\\(\\\\\\$\\)\\|${\\(\\([^}\\]\\|\\(\\\\.\\)\\)*\\)}" nil t)
```

Comment:
Could you move this regex to a defconst with a nice name?

Reply:
I would recommend to use `rx` for better maintenance

Reply:
I would love to use rx. Are we allowed to do that, as it is a dependency?

Reply:
`rx` is built-in. isn't it?